inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp

```diff
@@ -0,0 +1,448 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadBinaryConvolutionNetwork) {
```

Comment:
From my point test on the binary convolution should test only binary convolution.